cmd/cmd_test.go

```diff
@@ -134,6 +134,30 @@ func TestOptionsValidate(t *testing.T) {
 			}(),
 			"--no-follow cannot be used with --tail=0",
 		},
+		{
+			"Specify --condition without --tail=0 and no --no-follow",
+			func() *options {
+				o := NewOptions(streams)
+				o.podQuery = "."
+				o.condition = "ready=false"
+				o.tail = -1
+
+				return o
+			}(),
+			"--condition is currently only supported with --tail=0 or --no-follow",
+		},
+		{
+			"Specify --condition without --no-follow and no --tail=0",
+			func() *options {
+				o := NewOptions(streams)
+				o.podQuery = "."
+				o.condition = "ready=false"
+				o.noFollow = true
```

Comment:
```suggestion
```

This test case currently seems to fail. Should this be removed, as it is described as `without --no-follow`?

```
$ go test github.com/stern/stern/cmd -run TestOptionsValidate
--- FAIL: TestOptionsValidate (0.00s)
    --- FAIL: TestOptionsValidate/Specify_--condition_without_--no-follow_and_no_--tail=0 (0.00s)
        cmd_test.go:208: expected "--condition is currently only supported with --tail=0 or --no-follow" err, but actual no err
FAIL
FAIL    github.com/stern/stern/cmd      0.986s
FAIL
```

Reply:
ops.